src/perception/Perception.cpp

```diff
@@ -195,16 +195,9 @@ Eigen::Isometry3d Perception::perceiveFace()
       // fixed distance:
       double fixedFaceY
           = getRosParam<double>("/feedingDemo/fixedFaceY", *mNodeHandle);
-      if (fixedFaceY > 0)
-      {
-        faceTransform.translation().y() = fixedFaceY;
-        // Wheelchair
-        faceTransform.translation().z() -= 0.02;
+      faceTransform.translation().y() = fixedFaceY;
+      faceTransform.translation().z() -= 0.01;
```

Comment:
I'm guessing this is the out-of-mouth direction? Why do we adjust it by 1cm, given that we already have `distanceToPerson` as a param in the config file?

Reply:
This is not the out-of-mouth direction (that is addressed by `fixedFaceY`). This is the vertical direction (against gravity).

I believe the issue is that when we plan towards the stomion, a user would somehow need to move their head perfectly forward to get the food, but that's not how the neck (usually) works.

So we plan slightly *below* the stomion, so when the user bends their neck, they can more comfortably get the food.

Formalizing this as a parameter can be a separate issue.

Reply:
Ooh that makes a lot of sense! I think it could be useful to add a comment on this (for whoever next makes a push on this branch)